handbook/engineering/releases/release_issue_template.md

```diff
@@ -115,8 +124,6 @@ Once there are no more release-blocking issues (as reported by the `release:stat
 ### Post-release
 
 - [ ] Notify the next release captain that they are on duty for the next release. They should complete the steps in this section.
-- [ ] Ensure you have the latest version github.com/sourcegraph/about checked out and it is located
-      at `../about` (relative to this repository).
```

Comment:
as far as I can tell, this used to be for generating the issue templates, but we now pull it from GitHub